docs/GettingStarted.md

```diff
@@ -11,10 +10,19 @@ Pushed container '<your app name>:<your app version>' to registry 'docker://'
 ...
 ```
 
-Out of the box, this package will infer a number of properties about the generated container image, including which base image to use, which version of that image to use, and where to push the generated image. You have control over all of these properties, however. You can read more about customizing the container [here](./ContainerCustomization.md)
+For more information, see [Publish Profiles](https://docs.microsoft.com/aspnet/core/host-and-deploy/visual-studio-publish-profiles?view=aspnetcore-6.0#publish-profiles).
+
+The `Microsoft.NET.Build.Containers` package infers a number of properties about the generated container image:
+
+* Which base image to use.
+* Which version of that image to use.
+* Where to push the generated image.
+
+<!--The `Microsoft.NET.Build.Containers` package provides full control over the preceding properties. -->
```

Comment:
If you insist on keeping a replacement for `You have control over all of these properties, however.`, uncomment this line. I'm guessing that's not a complete list.